build.gradle

```diff
@@ -1,47 +1,50 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm' version '1.3.61'
-    id 'application'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72' apply false
 }
 
-repositories {
-    jcenter()
-    maven { url 'https://jitpack.io' }
-}
+subprojects {
+    apply plugin: 'org.jetbrains.kotlin.jvm'
```

Comment:
Do we need subprojects now (since we have only one module) or this is in case we have runtime module in future?

Reply:
Introducing ```runtime``` module soon :) Moving exception classes there